### ISSUE TITLE
exit code broken?

### ISSUE THREAD

**Issue by EYEBB** [arkmanager/ark-server-tools#963]
Hi,
  I was trying to get my cron on ran into what I think is a typo in the guide... or maybe the code.

```
[steam@ark-server ~]$ cat ark_nanny.sh
/usr/local/bin/arkmanager checkupdate @valhalla >> /home/steam/ark_nanny.log 2>&1
update=$?
/usr/local/bin/arkmanager checkmodupdate @valhalla >> /home/steam/ark_nanny.log 2>&1
modupdate=$?

echo $update
echo $modupdate
```
Which provides the following output:

```
[steam@ark-server ~]$ ./ark_nanny.sh 
0
1
```

However checking what's in the guide (README.asciidoc) the exit codes aren't matching up:

```
[steam@ark-server ~]$ cat ark_nanny.log 
Running command 'checkupdate' for instance 'valhalla'
Current version:  2893367 
Available version:  2893367 
Your server is up to date!
Running command 'checkmodupdate' for instance 'valhalla'
All mods are up to date
```

I then went ahead and added --revstatus to both just to see, and both returned '0' and '0'. 
I don't mind digging in code, but I wont be able to circle back around on this for awhile so I figured I'd document it here. PS love arkmanager, keep up the good work!

**Comment by cosmicc** [comment]
same thing here. it would be really nice to use exit codes instead of parsing the output.

**Comment by klightspeed** [comment]
I have now updated the documentation with the exit codes for `checkupdate`

**Comment by NoirPi** [comment]
Theres a really strange behaviour with these exit Codes:

```
/usr/local/bin/arkmanager checkupdate @valguero-pve >> /home/steam/test.log 2>&1
update=$?
echo ResponseCode: $? >> /home/steam/test.log
/usr/local/bin/arkmanager checkmodupdate --revstatus @valguero-pve >> /home/steam/test.log 2>&1
modupdate=$?
echo ResponseCode: $? >> /home/steam/test.log

echo ResponseCode: $update
echo ResponseCode: $modupdate
```

File Output in test.log:
> Running command 'checkupdate' for instance 'valguero-pve'
> ^[7Querying Steam database for latest version...
> ^[8^[[JCurrent version: ^[[1;32m 4143025 ^[[0;39m
> Available version: ^[[1;32m 4143025 ^[[0;39m
> Your server is up to date!
> ResponseCode: 0
> Running command 'checkmodupdate' for instance 'valguero-pve'
> All mods are up to date
> ResponseCode: 0


Console Output:
```
ResponseCode: 1
ResponseCode: 1
```

I've tried multiple Options to get a different result but it always ends in the same strange behaviour!

**Comment by klightspeed** [comment]
`$?` gives the status of the last operation.
Setting a variable will reset the status code.

```
$ false
$ x=$?
$ echo $?
0
$ echo $x
1
```

Try:
```
/usr/local/bin/arkmanager checkupdate @valguero-pve >> /home/steam/test.log 2>&1
update=$?
echo ResponseCode: $update >> /home/steam/test.log
/usr/local/bin/arkmanager checkmodupdate --revstatus @valguero-pve >> /home/steam/test.log 2>&1
modupdate=$?
echo ResponseCode: $modupdate >> /home/steam/test.log

echo ResponseCode: $update
echo ResponseCode: $modupdate
```

**Comment by NoirPi** [comment]
Yeah that changed the strange behaviour... Now i just get the wrong response code for the status of the Servers...

> Running command 'checkupdate' for instance 'valguero-pve'
> ^[7Querying Steam database for latest version...
> ^[8^[[JCurrent version: ^[[1;32m 4143025 ^[[0;39m
> Available version: ^[[1;32m 4143025 ^[[0;39m
> Your server is up to date!
> ResponseCode: 1
> Running command 'checkmodupdate' for instance 'valguero-pve'
> All mods are up to date
> ResponseCode: 1

**Comment by klightspeed** [comment]
Are you running as root?

If so, 40e373f should fix it.
